pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -519,6 +521,25 @@ var _ = Describe("Workload Updater", func() {
 			controller.Execute()
 			testutils.ExpectEvent(recorder, FailedChangeAbortionReason)
 		})
+
+		It("shouldn't cancel the migration if the migration object is still in running phase but the domain is ready on the target and the change condition has already been removed from the VMI", func() {
+			vmi := api.NewMinimalVMI("testvm")
+			vmi.Namespace = k8sv1.NamespaceDefault
+			vmi.Status.Phase = v1.Running
+			vmi.Status.Conditions = append(vmi.Status.Conditions, v1.VirtualMachineInstanceCondition{
+				Type: v1.VirtualMachineInstanceIsMigratable, Status: k8sv1.ConditionTrue})
+			vmi.Status.MigrationState = &v1.VirtualMachineInstanceMigrationState{
+				StartTimestamp:                 pointer.P(metav1.Now()),
+				Completed:                      false,
+				Failed:                         false,
+				TargetNodeDomainReadyTimestamp: pointer.P(metav1.Now()),
+			}
+			controller.vmiStore.Add(vmi)
+			createMig(vmi.Name, v1.MigrationRunning)
+			controller.Execute()
+			Expect(recorder.Events).To(BeEmpty())
```

Comment:
Can we leave a comment or explicitly use mock to express we do not expect `Delete` call?

Reply:
I can add a comment, if there is a call to Delete then the test will panic. Otherwise, I don't how to use mock to state that there wasn't that call. Any idea?

Reply:
You use `.Times(0)`

Reply:
And the reason why I ask for this is for us to not forget about this when we convert the tests to use fake client

Reply:
> You use `.Times(0)`

Aaah, nice trick

Reply:
Let me add with a comment as well